### PR TITLE
Add `--force` flag to `g8Scaffold` command

### PR DESCRIPTION
**Pull request by mkurz**
Flag to overwrite existing files.

- [x] Test
- [x] Docs (english)

What do you think? IMHO nothing complicated.

### CODE REVIEW

File: docs/04/00.md [hunk 1]
```diff
@@ -30,6 +30,8 @@ The template plugin will prompt each property that needed to complete the scaffo
 className [Application]:
 ```
 
+The `g8Scaffold` command will not overwrite existing files.
```
Comment (mkurz) [inline docs/04/00.md]: Someone wants to translate that to [Japanese](https://github.com/foundweekends/giter8/blob/develop/docs/ja/04/00.md) and/or [Korean](https://github.com/foundweekends/giter8/blob/develop/docs/ko/04/00.md)?

**Comment by mkurz** [comment]
The `g8` command already has the `--force` flag, so I named it "force" too: https://github.com/foundweekends/giter8/blob/d8ef0a0cad53e2e2481b4c0fe8ef3ded1fcb899e/docs/02.md?plain=1#L60-L63

**Comment by mkurz** [comment]
@xuwei-k / @eed3si9n Grateful if you could take a look at this PR and #774 (and #775) and hopefully merge it...
Would be awesome if you could cut a new release :pray: Thanks!

**Comment by mkurz** [comment]
If you want I can rename to `g8ForceScaffold` if that would be a better naming.

File: scaffold/src/main/scala/ScaffoldPlugin.scala [hunk 2]
```diff
@@ -46,19 +47,23 @@ object ScaffoldPlugin extends sbt.AutoPlugin {
           }
     }
 
-  val scaffoldTask =
-    Def.inputTask {
-      val (name, args) = parser.parsed
-      val folder       = g8ScaffoldTemplatesDirectory.value
-      G8.fromDirectoryRaw(folder / name, baseDirectory.value, args, false)
```
Comment (eed3si9n) [inline scaffold/src/main/scala/ScaffoldPlugin.scala:53]: If `g8Scaffold` is an input task, would it make sense to parse `--force` in `G8.fromDirectoryRaw` somehow?

Comment (mkurz) [reply scaffold/src/main/scala/ScaffoldPlugin.scala:53]: @eed3si9n Changed to use the `--force` flag. Can you take a look again? Thanks!

**Review by eed3si9n — APPROVED** [review]
Thanks!

**Comment by mkurz** [comment]
Thanks! Would you mind releasing my recent fixes?

**Comment by mkurz** [comment]
Any chance for a new release soon? Thanks!

**Pull request merged**